src/scripts/main.js

```diff
@@ -1,3 +1,38 @@
 'use strict';
 
-// write code here
+const employeeList = document.querySelector('ul');
+
+function parseSalary(salaryString) {
+  const signFree = salaryString.split('$').join('');
+  const commaFree = signFree.split(',').join('');
+
+  return Number(commaFree);
+}
```

Comment:
The `parseSalary` function assumes that the salary string will always contain a dollar sign and commas. Consider adding validation to handle cases where these might be missing or malformed.